DuckDuckGo/ViewExtension.swift

```diff
@@ -77,6 +77,12 @@ extension View {
             .applyBackground()
     }
 
+    /// Removes the grouped list style insets for a single row.
+    ///
+    func removeGroupedListStyleInsets() -> some View {
+        listRowInsets(EdgeInsets(top: 0, leading: 0, bottom: 0, trailing: 0))
+    }
```

Comment:
Convenience method to style tip views in a way they can be easily embedded in a list while following the style of other cells.